app/cdash/app/Model/Build.php

```diff
@@ -1115,6 +1116,17 @@ private function GetTests($criteria, $maxitems = 0)
         return $query->fetchAll(PDO::FETCH_ASSOC);
     }
 
+    /**
+     * Get this build's tests that passed.
+     *
+     * @return array
+     */
+    public function GetPassedTests($maxitems = 0)
```

Comment:
It is generally better to use PHP's native type hinting when possible, since annotations cannot be enforced by the interpreter.  More complex things such as "an array of strings" can be specified via an annotation to help PHPStan understand things a bit more.

Apply type hints throughout these changes.

```suggestion
    /** Get this build's tests that passed. */
    public function GetPassedTests($maxitems = 0): array
```